### PR TITLE
Update media-dynamic-update.md

### PR DESCRIPTION
**Pull request by SteveDiAcetis**
Additional information on Flash removal

### CODE REVIEW

**Comment by SteveDiAcetis** [comment]
@jaimeo please review, and let me know if you have any suggested edits. @sudhagart , if you have time, also please review.

**Comment by jaimeo** [comment]
@SteveDiAcetis did you really mean versions 1607 and 1507? Just double-checking. I made just a few tweaks for style.

**Comment by SteveDiAcetis** [comment]
@jaimeo Thanks, yes, I used the same text from the flash end of support blog. I'm good with your edits. I suggest we give this a couple of days to let Sudha review, then you can merge.

File: windows/deployment/update/media-dynamic-update.md [hunk 1]
```diff
@@ -84,6 +84,9 @@ This table shows the correct sequence for applying the various tasks to the file
 > [!NOTE]
 > Starting in February 2021, the latest cumulative update and servicing stack update will be combined and distributed in the Microsoft Update Catalog as a new combined cumulative update. For Steps 1, 9, and 18 that require the servicing stack update for updating the installation media, you should use the combined cumulative update. For more information on the combined cumulative update, see [Servicing stack updates](./servicing-stack-updates.md).
 
+> [!NOTE]
+> Microsoft will remove the Flash component from Windows through KB4577586, “Update for Removal of Adobe Flash Player”. You can also remove Flash anytime by deploying the update in KB4577586 (available on the Catalog) between steps 20 and 21. As of July 2021, KB4577586, “Update for Removal of Adobe Flash Player,” will be included in the latest cumulative update for Windows 10, versions 1607 and 1507. The update will also be included in the Monthly Rollup and the Security Only Update for Windows 8.1, Windows Server 2012, and Windows Embedded 8 Standard. For more information, see [Update on Adobe Flash Player End of Support](https://blogs.windows.com/msedgedev/2020/09/04/update-adobe-flash-end-support/).
```
Comment (JohanFreelancer9) [inline windows/deployment/update/media-dynamic-update.md:88]:
```suggestion
> Microsoft will remove the Flash component from Windows through KB4577586, “Update for Removal of Adobe Flash Player”. You can also remove Flash anytime by deploying the update in KB4577586 (available on the Catalog) between steps 20 and 21. As of July 2021, KB4577586, “Update for Removal of Adobe Flash Player”, will be included in the latest cumulative update for Windows 10, versions 1607 and 1507. The update will also be included in the Monthly Rollup and the Security Only Update for Windows 8.1, Windows Server 2012, and Windows Embedded 8 Standard. For more information, see [Update on Adobe Flash Player End of Support](https://blogs.windows.com/msedgedev/2020/09/04/update-adobe-flash-end-support/).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SteveDiAcetis, please commit the minor suggested change, please.

**Comment by jaimeo** [comment]
I don't know who moved the comma, but commas (and other punctuation) go INSIDE quotation marks. But I removed the comma altogether anyway, because one doesn't belong there.

**Pull request merged**